tests/tensor_search/test_multimodal_tensor_combination.py

```diff
@@ -0,0 +1,289 @@
+import unittest.mock
```

Comment:
1. Can we get the tensor_facets (by get_document(show_vectors=True)) and assert they are correct? 
2. Can we get the original documents from Marqo-OS and do assertions on them? Is the meta data correct?

Reply:
done